app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [
+        Person(person_data["name"], person_data["age"])
+        for person_data in people
+    ]
+    for person_data in people:
```

Comment:
Does `person_data` is the same?  looks like not, because at first situation it's a dict, in a second it instance of Person class.
Make better naming at that situation